From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Wayne Chang <waynec@nvidia.com>
Date: Thu, 14 Nov 2024 03:11:36 +0000
Subject: [PATCH 161/254] NVIDIA: SAUCE: soc/tegra: fuse: Add
 FUSE_USB_CALIB_EXT2 and FUSE_USB_CALIB_EXT3

Add FUSE_USB_CALIB_EXT2 and FUSE_USB_CALIB_EXT3 which are used for USB
calibration.

Bug 4933929

Change-Id: Ie6123ab698733cc4ee95370dcdcdb6492d6444f0
Signed-off-by: Wayne Chang <waynec@nvidia.com>
Reviewed-on: https://git-master.nvidia.com/r/c/3rdparty/canonical/linux-noble/+/3248457
Reviewed-by: Henry Lin <henryl@nvidia.com>
GVS: buildbot_gerritrpt <buildbot_gerritrpt@nvidia.com>
Reviewed-by: Sing-Han Chen <singhanc@nvidia.com>
Reviewed-by: WK Tsai <wtsai@nvidia.com>
Reviewed-by: HaoTien Hsu <haotienh@nvidia.com>
(cherry picked from commit 848fc623c0751a3b762488b0c30a0ea94853f0b2)

diff --git a/include/soc/tegra/fuse.h b/include/soc/tegra/fuse.h
index 8f421b9f7585c..5a112763457aa 100644
--- a/include/soc/tegra/fuse.h
+++ b/include/soc/tegra/fuse.h
@@ -23,6 +23,8 @@
 #define TEGRA_FUSE_SKU_CALIB_0	0xf0
 #define TEGRA30_FUSE_SATA_CALIB	0x124
 #define TEGRA_FUSE_USB_CALIB_EXT_0 0x250
+#define TEGRA_FUSE_USB_CALIB_EXT2_0 0x2ec
+#define TEGRA_FUSE_USB_CALIB_EXT3_0 0x3bc
 
 #ifndef __ASSEMBLY__
 
-- 
2.43.0

